modules/dataframe/include/inviwo/dataframe/datastructures/dataframe.h

```diff
@@ -240,13 +241,59 @@ struct DataTraits<DataFrame> {
         tb(H("Number of Rows: "), data.getNumberOfRows());
 
         utildoc::TableBuilder tb2(doc.handle(), P::end());
-        tb2(H("Col"), H("Format"), H("Rows"), H("Name"));
+        tb2(H("Col"), H("Format"), H("Rows"), H("Name"), H("Min"), H("Max"));
         // abbreviate list of columns if there are more than 20
         const size_t ncols = (data.getNumberOfColumns() > 20) ? 10 : data.getNumberOfColumns();
 
-        for (size_t i = 0; i < ncols; i++) {
+        if (ncols < 2) return doc;
+
+        for (size_t i{1}; i < ncols; i++) {
+            auto col = data.getColumn(i);
+
+            auto [minString, maxString] =
+                col->getBuffer()
+                    ->getRepresentation<BufferRAM>()
+                    ->dispatch<std::pair<std::string, std::string>, dispatching::filter::Scalars>(
+                        [](auto brprecision) -> std::pair<std::string, std::string> {
+                            using ValueType = util::PrecisionValueType<decltype(brprecision)>;
+
+                            const auto &vec = brprecision->getDataContainer();
+
+                            if (vec.empty()) {
+                                return {"-", "-"};
+                            }
+
+                            if constexpr (std::is_floating_point_v<ValueType>) {
```

Comment:
These two tests are unnecessary, the `stringstream` handles them correctly. Especially in the case of `inf`, where one of them can be valid when the other is neg-inf/inf.  
These tests were only needed before when substring based on the dot location was used.

Reply:
True. I also noticed that this will throw a runtime error if the data frame contains vector types. I'll fix that soon.